package.json

```diff
@@ -4,10 +4,12 @@
   "description": "Node.js-compatible utility functions for Bare",
   "exports": {
     "./package": "./package.json",
-    ".": "./index.js"
+    ".": "./index.js",
+    "types": "./types.js"
```

Comment:
@yassernasc Should this be `./types` instead? Isn't `types` a convention for exporting Typescript types?